firmware/user/display/oled.c

```diff
@@ -139,7 +139,7 @@ bool fbChanges = false;
  */
 void ICACHE_FLASH_ATTR clearDisplay(void)
 {
-    ets_memset(currentFb, 0, sizeof(currentFb));
+    ets_memset(currentFb, 0, (OLED_WIDTH * (OLED_HEIGHT / 8)) );
```

Comment:
🍠🥔

Reply:
Sorry, this was left over from when I was actually using pointers to deal with the buffers.  I flipped back.